_components/identifier/guidance/usability.md

```diff
@@ -1,4 +1,5 @@
 - **Use the identifier component for required links.** If your site already includes the federally required links in its site footer, remove them in favor of the links in the identifier. This assures that site visitors find the required links in a consistent location from site to site.
+- **Consider the parent agency to be the highest-level agency associated with a site or service.** In some cases, your site may not have a parent agency. If your site is the primary site for your agency, use your agency name in place of [Parent Agency]. For example, "[agency.gov] An official website of [Agency]." 
```

Comment:
Recommend removing "to be" from the first sentence. ("Consider the parent agency the highest-level...")

Reply:
```suggestion
- **Consider the parent agency the highest-level agency associated with a site or service.** In some cases, your site may not have a parent agency. If your site is the primary site for your agency, use your agency name in place of [Parent Agency]. For example, "[agency.gov] An official website of [Agency]." 
```